symplyphysics/core/vectors/vector_arithmetics.py

```diff
@@ -0,0 +1,49 @@
+from sympy import Expr
+from .vectors import Vector
+from ..expr_comparisons import expr_equals
+
+
+## Only works for Cartesian coordinates!
+#TODO: rework to support other coordinate systems.
+#TODO: add vector_length implementation
+
+# Compare two Python vectors
+def equal_vectors(vector_left: Vector, vector_right: Vector) -> bool:
+    if vector_left.coord_system != vector_right.coord_system:
+        raise TypeError(f"Different coordinate systems in vectors: {str(vector_left.coord_system)} vs {str(vector_right.coord_system)}")
+    for i in range(max(len(vector_left.components), len(vector_right.components))):
+        val1 = 0 if i >= len(vector_left.components) else vector_left.components[i]
+        val2 = 0 if i >= len(vector_right.components) else vector_right.components[i]
+        if not expr_equals(val1, val2): return False
+    return True
+
+# Sum of two Python vectors
+# Sum of two vectors can be seen as a diagonal of the parallelogram, where vectors are adjacent sides of this parallelogram.
+# To subtract vectors, multiply one of the vectors to -1 and add them.
+def add_vectors(vector_left: Vector, vector_right: Vector) -> Vector:
+    if vector_left.coord_system != vector_right.coord_system:
+        raise TypeError(f"Different coordinate systems in vectors: {str(vector_left.coord_system)} vs {str(vector_right.coord_system)}")
+    result = []
+    for i in range(max(len(vector_left.components), len(vector_right.components))):
+        val1 = 0 if i >= len(vector_left.components) else vector_left.components[i]
+        val2 = 0 if i >= len(vector_right.components) else vector_right.components[i]
+        result.append(val1 + val2)
+    return Vector(result, vector_left.coord_system)
+
+# Scalar Python vector multiplication
+# Scalar multiplication changes the magnitude of the vector and does not change it's direction.
```

Comment:
May we name this operation "magnification" or "scaling"? Just not to cobfuse it with scalar operation vector dot vector?